### ISSUE TITLE
[Commands] [Scarpet] Port `/info` to a Scarpet app

### ISSUE THREAD

**Issue by altrisi** [gnembon/fabric-carpet#1386]
Scarpet apps are more maintainable given they don't depend on the game source directly (so only the language has to be updated once for all apps to work) and having Carpet functionality as scarpet apps also works as tests for the scarpet API.

I propose `/info` to be reimplemented as a Scarpet app, given it is quite a simple concept if scarpet has access to all the data `/info` provides, and if it doesn't, this can be a good tracking issue for adding them.

**Comment by gnembon** [comment]
yes. This just shows the gap in the API. There are more 'exotic' measures, like random wandering chances, but that could be measured in scanning the area for blocks, so not a biggie..

Another vector would be informative aspects of /spawn - could also be scarpet only.
